CSS/portfolio.css

```diff
@@ -354,5 +354,10 @@ body {
     color: aquamarine;
 }
 
+.login {
+    color: floralwhite;
```

Comment:
change colors to hex or rgb() values for more consistency and support across browsers